infra/cluster-config-setup.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
note: this is very similar conceptually to the installer's config bootstrapping process

I also added most of the config items which were previously places by the CDK sub-stacks that get invoked by the installer stack